src/components/TodoListItem.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const TodoListItem = ({toDo}) => {
```

Comment:
:100: for destructuring the prop!

Reply:
Thank you! I had to read throughout the extra links the book gives us in order to learn object destructuring!